### ISSUE TITLE
Question about forward_train()

### ISSUE THREAD

**Issue by jetitime** [lhoyer/MIC#70]
Hi @lhoyer @krumo,
    Thanks for your great work in UDA segmentation, and this helps my study.  I have some questions while reading the code, in the dacs.py file you can calculate the cross entropy loss directly using get_model().forward_train(). But I can't quite find the specific form of gt_semantic_seg, such as whether it has been softmax processed, etc. What if I want to obtain a concept map of segmentation prediction? Hope you can give me some guidance on where I should look at the detailed process of forward training. Thank you!
